src/components/modals/AddressModal.js

```diff
@@ -11,16 +11,20 @@ import * as Constants from '../../constants/indexConstants'
 import s from '../../locales/strings.js'
 import styles from '../../styles/scenes/ScaneStyle'
 import { colors as COLORS } from '../../theme/variables/airbitz.js'
+import ResolutionError, { ResolutionErrorCode} from '../common/ResolutionError.js'
 
 // INTERACTIVE_MODAL /////////////////////////////////////////////////////////////////////////////
 type AddressModalProps = {
   onDone: any => void,
-  coreWallet: EdgeCurrencyWallet
+  coreWallet: EdgeCurrencyWallet,
+  currencyCode: string
```

Comment:
unused?

Reply:
This is needed for the error code in cases when some domain doesn't have a certain ticker configured. 

Example of the error message would be: 
" Domain brad.crypto has no BTC address attached to it "